deploy/kubernetes/manifests/csidriver.yaml

```diff
@@ -3,7 +3,6 @@ kind: CSIDriver
 apiVersion: storage.k8s.io/v1
 metadata:
   name: vpc.file.csi.ibm.io
-  namespace: openshift-cluster-csi-drivers
```

Comment:
@prankulmahajan is this expected ?

Reply:
its not a namespaced resource. So its not required